tests/lib/hardware/openstack_libcloud.py

```diff
@@ -313,20 +313,25 @@ def _create_security_group(self):
             )
         return security_group
 
-    def _create_node(self, node_name, role, tags=[]):
+    def node_create(self, name: str, role: NodeRole,
+                    tags: List[str]) -> NodeBase:
+        super().node_create(name, role, tags)
         # are there any additional networks for the node wanted?
         additional_networks = []
         if config.OS_INTERNAL_NETWORK:
             additional_networks.append(
                 self._get_ex_network_by_name(config.OS_INTERNAL_NETWORK)
             )
 
-        node = Node(node_name, role, tags, self.conn,
+        node = Node(name, role, tags, self.conn,
                     self._get_size_by_name(config.NODE_SIZE),
                     self._get_image(config.NODE_IMAGE),
                     additional_networks, [self._ex_security_group],
                     self.workspace.sshkey_name)
+        return node
 
+    def _create_node(self, node_name, role, tags=[]):
```

Comment:
it's not important since this is an internal method, but for consistency should this be _boot_node?
Nevertheless it's the same as before so can be followed up.